config.go

```diff
@@ -66,16 +68,57 @@ func LoadConfigurationFile(path string) (Config, error) {
 	return conf, err
 }
 
+func GetOSConfigPath(homePath string) string {
+	var configDir string
+
+	defaultConfig := map[string]string{
+		"windows": "LocalAppData",
+		"darwin":  "Library/Application Support",
+		"linux":   ".config",
+	}
+
+	goos := runtime.GOOS
+	switch goos {
+	case "windows":
+		configDir = os.Getenv("LOCALAPPDATA")
+	case "darwin", "linux":
+		configDir = os.Getenv("XDG_CONFIG_HOME")
+	case "default":
+		// exit with no path for unknown os
+		return ""
+	}
+
+	if configDir == "" {
+		configDir = filepath.Join(homePath, defaultConfig[goos])
+	}
+
+	return filepath.Join(configDir, "eget", "eget.toml")
+}
```

Comment:
Ya I saw this library and since we only needed it for `XDG_CONFIG_HOME` and not any other `XDG_BASE_DIRS` I opted to write a small function instead. But I could refactor to just use this library if that make more sense.